tests/foreman/cli/test_errata.py

```diff
@@ -60,16 +72,130 @@
     run_in_one_thread,
     run_only_on,
     skip_if_bug_open,
+    skip_if_not_set,
 )
 from robottelo.test import CLITestCase
+from robottelo.vm import VirtualMachine
 
 ERRATUM_MAX_IDS_INFO = 10
 
 
+@skip_if_bug_open('bugzilla', 1405428)
+@run_in_one_thread
 class HostCollectionErrataInstallTestCase(CLITestCase):
     """CLI Tests for the errata management feature"""
 
-    @stubbed()
+    CUSTOM_REPO_URL = FAKE_6_YUM_REPO
+    CUSTOM_PACKAGE = FAKE_1_CUSTOM_PACKAGE
+    CUSTOM_ERRATA_ID = FAKE_2_ERRATA_ID
+    CUSTOM_PACKAGE_ERRATA_APPLIED = FAKE_2_CUSTOM_PACKAGE
+    VIRTUAL_MACHINES_COUNT = 2
+
+    @classmethod
+    @skip_if_not_set('clients', 'fake_manifest')
+    def setUpClass(cls):
+        """Create Org, Lifecycle Environment, Content View, Activation key,
+        Host, Host-Collection
+
+        """
+        super(HostCollectionErrataInstallTestCase, cls).setUpClass()
+        cls.org = make_org()
+        cls.env = make_lifecycle_environment({
+            'organization-id': cls.org['id']
+        })
+        cls.content_view = make_content_view({
+            'organization-id': cls.org['id']
+        })
+        cls.activation_key = make_activation_key({
+            'lifecycle-environment-id': cls.env['id'],
+            'organization-id': cls.org['id']
+        })
+        # add subscription to Satellite Tools repo to activation key
+        setup_org_for_a_rh_repo({
+            'product': PRDS['rhel'],
+            'repository-set': REPOSET['rhst7'],
+            'repository': REPOS['rhst7']['name'],
+            'organization-id': cls.org['id'],
+            'content-view-id': cls.content_view['id'],
+            'lifecycle-environment-id': cls.env['id'],
+            'activationkey-id': cls.activation_key['id']
+        })
+        # create custom repository and add subscription to activation key
+        setup_org_for_a_custom_repo({
+            'url': cls.CUSTOM_REPO_URL,
+            'organization-id': cls.org['id'],
+            'content-view-id': cls.content_view['id'],
+            'lifecycle-environment-id': cls.env['id'],
+            'activationkey-id': cls.activation_key['id']
+        })
+
+    def _create_host_collection(self):
```

Comment:
Redundant helper which is called only once. I get the idea in current implementation, but when you switch to cleanup funcs, both `_create_host_collection` and `_destroy_virtual_machines` will become completely redundant, so pls remove them.